src/issue-filing/rule-result-to-issue-filing-data.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
There seems no unit test for this class?

Reply:
Ah, thanks! mixed up my branches - let me port it over